builder/package.ubuntu-1604

```diff
@@ -33,6 +33,7 @@ fpm \
   -d libpango-1.0-0 \
   -d libpangocairo-1.0-0 \
   -d libpaper-utils \
+  -d libpcre2 \
```

Comment:
This should be `libpcre2-8` (and ditto for Ubuntu 18). But otherwise, this looks good to me.

Reply:
oops: I meant `libpcre2-8-0`